packages/terra-list/tests/nightwatch/common.js

```diff
@@ -0,0 +1,18 @@
+function checkSingleElementState(browser, selector, className, isPresent) {
+  if (isPresent) {
+    browser.expect.element(selector).to.have.attribute('class').which.contains(className);
+  } else {
+    browser.expect.element(selector).to.have.attribute('class').which.not.contains(className);
+  }
+}
+
+function checkElementsClass(browser, selectors = [], classes = [], states = []) {
```

Comment:
This is the function commonly used in nightwatch test to check appropriate class present or not. [link](https://github.com/cerner/terra-core/pull/648/files#diff-eff6a773f470bd9311106c36708c22e5R62)